presto-main/src/test/java/io/prestosql/sql/analyzer/TestAnalyzer.java

```diff
@@ -1639,58 +1639,53 @@ public void setup()
                 false));
 
         // valid view referencing table in same schema
-        String viewData1 = JsonCodec.jsonCodec(ViewDefinition.class).toJson(
-                new ViewDefinition(
-                        "select a from t1",
-                        Optional.of(TPCH_CATALOG),
-                        Optional.of("s1"),
-                        ImmutableList.of(new ViewColumn("a", BIGINT)),
-                        Optional.of("user"),
-                        false));
+        ConnectorViewDefinition viewData1 = new ConnectorViewDefinition(
+                "select a from t1",
+                Optional.of(TPCH_CATALOG),
+                Optional.of("s1"),
+                ImmutableList.of(new ViewColumn("a", parseTypeSignature("bigint"))),
```

Comment:
use this style `BIGINT.getTypeSignature()` for all of these instead of `parseTypeSignature`

Reply:
Done